test/Point.java

```diff
@@ -2,6 +2,10 @@
 
 /**
  * An immutable point in floating-point pixel space.
+ * AF (x,y)= a point with the double x as the  x-coordinate,double y as the  y-coordinate
+ * Representation invariant:
+ * Safety from rep exposure:
+ * All fields are private final;
```

Comment:
The description of the safety from rep exposure is added. That's good.

Reply:
You remind me that the represention invariant of this **point** object is not completed yet, but I'll complete it later. Thanks.